test/YesSql.Tests/CoreTests.cs

```diff
@@ -1966,19 +1966,19 @@ await using (var session = _store.CreateSession())
             await using (var session = _store.CreateSession())
             {
                 Assert.Equal(2, await session.Query().For<Person>()
-                    .With<PersonByName>(x => x.SomeName.StartsWith("S"))
```

Comment:
This is not supported, only StartsWith(string) is bound

Reply:
This could be added, but will require a custom test too, see `static DefaultQuery()`